sdk/sdktypes/integration_id.go

```diff
@@ -22,8 +23,33 @@ func StrictParseIntegrationID(s string) (IntegrationID, error) { return Strict(P
 
 func IsIntegrationID(s string) bool { return IsIDOf[integrationIDTraits](s) }
 
-func IntegrationIDFromName(name string) IntegrationID {
-	txt := fmt.Sprintf("%s_%026x", integrationIDKind, kittehs.HashString64(name))
+const integrationIDFromNamePrefix = "3kth"
```

Comment:
I still think this prefix isn't useful. It's obvious that this kind of ID is special and looks different from regular UUIDs, and while you see "❤️🐱", I see "3000th".

Perosnally, I think it's better to use 0's to separate the name and the hash, i.e. `"<prefix> _ 00 %06s 00 %016s"`

But it's a stylistic choice, so I won't block on it.

Reply:
you can't put 0 in the beginning of an id, uuid complains. so i chose 3. i needed some more chars to make it a bit more distinguishable than a random 0000.